.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,112 @@
+name: Bug Report
+description: Help us diagnose and fix bugs in Official AWS Provider
+labels: bug,needs:triage
+title: "[Bug]: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        # Thank you for helping to improve Official AWS Provider!
+
+        - If you're not currently using the most recent version of the Upbound Official AWS Provider, we recommend upgrading. This could resolve your issue, as any potential issues may have already been addressed in the latest release.
+
+  - type: checkboxes
```

Comment:
I didn't add an `id` here because there is an element of type checkboxes.

The absence of the `id` will not cause an issue of functionality but I'm adding it to define the element more specifically.